### PR TITLE
Add blog post on effects of DAO funding model

### PR DESCRIPTION
**Pull request by m52go**
To partially address https://github.com/bisq-network/growth/issues/97.

There's a lot to talk about on that topic so I will cover it over a short series of posts. This is the first one.

### CODE REVIEW

**Review by ManfredKarrer — APPROVED** [review]
utACK

It is a bit hard to read the final version with several commits, but I trust the end result is good ;-) The preview with netlify does not work with the blog post.

**Comment by ManfredKarrer** [comment]
@m52go I leave merging to yourself if you want to sync it with other activities...

**Comment by m52go** [comment]
@ManfredKarrer you can see the preview here:

https://deploy-preview-149--bisq-website.netlify.com/blog/dao-benefits-funding/

File: _posts/2019-03-13-dao-benefits-funding.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+<hr>
+
+At the [MIT Bitcoin Expo](https://mitbitcoinexpo.org/) this past weekend, funding open-source software development was a topic of high interest. Arjun Balaji's whole talk was focused on the topic, and a number of people approached me after my panel discussion wondering how the Bisq DAO worked to fund the project without any kind of external fundraising.
+
+The Bisq network's approaches to the challenges of funding open-source software and governing a revenue-earning operation are novel, and I had a great time discussing them with people at the event.
+
+In this post, we'll cover the practical _results_ of these approaches from a financial perspective: how an integrated revenue model—coupled with decentralized governance—results in an open-source software project that can grow and prosper on its own, while actually being _financially incentivized_ to protect its users' privacy.
+
+Future posts in this series will compare the dynamics among users and contributors in the Bisq network to their counterparts in the regular business world, and show how the Bisq DAO discourages
```
Comment (devinbileck) [inline _posts/2019-03-13-dao-benefits-funding.md]: Is this incomplete?

Comment (m52go) [reply _posts/2019-03-13-dao-benefits-funding.md]: @devinbileck yes, that's not right. thanks! fixed with most recent commit.

**Pull request merged**